### ISSUE TITLE
User notifications not following user choices

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1042]
Reported by Les at BC who has apparently had a number of complaints:
"Is there an issue in iRecord with switching off Email digest frequency for automated Record Cleaner record checks? I have mine set to Never but I still receive emails with records that fail the (old) rules."

**Comment by kitenetter** [comment]
I've had another report of this, from a verifier who has all notifications set to "never" but is definitely receiving the "your have records to verify" notifications (not sure if they are also receiving the others).

Moving into the milestone as this is causing annoyance.

**Comment by kitenetter** [comment]
This issue seems to have arisen following the changes in #939

**Comment by johnvanbreda** [comment]
If I masquerade as Les on iRecord, the "Email digest frequency for automated Record Cleaner record checks" setting is Daily. Are you sure he had switched this off?

@kitenetter Please can you let me know which verifier recieved a notification about records to verify when this setting was disabled, and when if possible.

**Comment by kitenetter** [comment]
@johnvanbreda the verifier who is definitely receiving "records to verify" notifications is:
warehouse user ID 102855
https://www.brc.ac.uk/irecord/user/23209/edit

I will check with Les. I think he was reporting that other county moth recorders had had similar problems.

**Comment by kitenetter** [comment]
Just found that Les has two accounts, one of which has the record cleaner notifications switched off, and the other doesn't, which probably explains his confusion. Will see if I can find out who else has been complaining.

**Comment by johnvanbreda** [comment]
@kitenetter I've looked at the data for this user in the database: https://www.brc.ac.uk/irecord/user/23209/edit. They had a notification setting for verifier notifications set to Daily, but this setting was removed on the 16th April, the same day you posted. So, up to that point I think it was correct that this user received notifications, but not after. Running the queries just now which pick up new verifier notifications, I can't see how they would receive a "New records to verify" notification, unless a priority verification species identified in the workflow setup, e.g. Asian Hornet.
